docs/using/administering/debugging.rst

```diff
@@ -33,6 +35,46 @@ It is possible to see the available unit names, and then view the logs with ``jo
    [root@node1]# journalctl -u flocker-container-agent
    [root@node1]# journalctl -u flocker-control
 
+Bug Reporting
+-------------
+
+When reporting issues with Flocker please include:
+
+* Operating System and version,
+* kernel version,
+* Docker info and version,
+* all Flocker log files.
+
+The following scripts can be used to gather this information on Ubuntu and CentOS.
+
+Save the appropriate script to a file and then run it as root
```

Comment:
I would prefer this instruction to appear between the download file, and the prompt in each of the sub-sections:

For example, 
Save the script to a file and then run the following as root: